src/lib/serviceProfiles.js

```diff
@@ -0,0 +1,13 @@
+/** values for profile that indicate an image service */
+const iiifImageProfiles = [
+  'level2',
+  'level1',
+  'level0',
+  'http://iiif.io/api/image/2/level2.json',
```

Comment:
I think there's an even older set of image profile urls, e.g.:

http://library.stanford.edu/iiif/image-api/compliance.html#level1